### DISCUSSION TITLE
XML export

### DISCUSSION THREAD

**Discussion by vgenov-py** [pawelsalawa/sqlitestudio#4902] [Q&A]
Hallo Paweł Salawa!

Pleasure to say hi!

Hey, this is actually not an issue, but I don't know how to communicate with you if it's not through this via.

Is it possible that you can tell me and the community where exactly can we find the xml export behind the scenes C function?

It's just that as you may know, there's not an XML export implemented flag on the SQLite CLI.

And it would be nice to do something like
`bash
sqlite3 your.db 'SELECT * FROM dogs' --xml > db.xml`

If you can great! if not also!

Kind regards!

**Comment by pawelsalawa** [comment]
https://github.com/pawelsalawa/sqlitestudio/blob/master/Plugins/XmlExport/xmlexport.cpp
